### PR TITLE
WIP Add ops manual

### PR DESCRIPTION
**Pull request by 0atman**

### CODE REVIEW

File: ops_manual.md [hunk 1]
```diff
@@ -5,30 +5,12 @@
...
 #### Architecture
-> TODO
 ```
-* A [Route53](https://aws.amazon.com/route53/) record pointing to ...
-* a [CloudFront](https://aws.amazon.com/cloudfront/) CDN which routes to ...
-* an [API Gateway](https://aws.amazon.com/api-gateway/) which routes to ...
-* some [Lambdas](https://aws.amazon.com/lambda/) which connect to ...
-* A [VPC](https://aws.amazon.com/vpc/) via an [ENI](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html) which allow it to connect to ...
-* A PostgreSQL [RDS](https://aws.amazon.com/rds/) instance in the VPC.
+* A [Lambdas](https://aws.amazon.com/lambda/) which connect to GitHub and Splunk
```
Comment (pritchyspritch) [inline ops_manual.md]: No need to the 'A' here and should we link the GitHub to Alphagov URL?

File: ops_manual.md [hunk 2]
```diff
@@ -99,42 +79,26 @@ To view the pipeline you need to login and have access to the
...
 To create an environment follow the instructions in the readme.
 
 ### How does it work?
 
-The infrastructure is terraformed
-(from the [csw-infra](https://github.com/alphagov/csw-infra) repository).
-
-As part of the ci process. Terraform deploys the VPC, security groups,
-subnets, routes etc. Merge your PR and it will be done.
+The infrastructure is terraformed from the terraform in the terraform directory.
```
Comment (pritchyspritch) [inline ops_manual.md]: terraform terraform terraform...terraform?

Comment (akinnane) [reply ops_manual.md]: Buffalo buffalo Buffalo buffalo buffalo buffalo Buffalo buffalo!

File: ops_manual.md (hunk 1 above)
Comment (akinnane) [reply ops_manual.md]: Fixed

**Pull request merged**